### DISCUSSION TITLE
OAuth 2.0 Bearer Token authentication only

### DISCUSSION THREAD

**Discussion by billbellon** [danieldevine/bird-elephant#29] [Q&A] [answered]
Can I use this package to retrieve public Tweets from a specific Twitter account as outlined here:

https://developer.twitter.com/en/docs/tutorials/authenticating-with-twitter-api-for-enterprise/authentication-method-overview#oauth2.0

Specifically, do I need the token_identifier and token_secret?  I don't have these for the Twitter account I want to retrieve public Tweets from and the process to get these 

https://developer.twitter.com/en/docs/authentication/oauth-1-0a/obtaining-user-access-tokens

looks rather involved.

**Comment by danieldevine** [comment, marked as answer]
Bird Elephant makes it pretty trivial to access another user's public tweets and information, i recommend that you read the docs on the subject: https://birdelephant.com/users#twitter-user 

You just need to follow the example in the first link you posted and get your "Consumer Keys (API Key and Secret) or a Bearer Token" for _your own account_. The content of the second link isn't relevant for what you want to do.

If you follow the documentation it tells you what auth you'll need to use for each endpoint. Methods marked as 'Auth: OAuth 2.0 Bearer token' can all be accessed with just your bearer token.
